### DISCUSSION TITLE
Branding Brainstorming

### DISCUSSION THREAD

**Discussion by WillieCubed** [UTDNebula/website#2] [Ideas]
Just to keep ideas about logos, colors, and theme stuff in one place and track development, let's open a discussion.

Feel free to drop any inspiration or ideas here!

The first big question is "What do you think of when you think of Nebula?"

**Comment by litiwu** [comment]
![logo_sketch](https://user-images.githubusercontent.com/59784432/124054214-2652ca00-da54-11eb-81a5-d1f0dd1c963d.jpg)

When I think of nebula I think of the swirly thing in the galaxy. I wanted to incorporate that image into the logo. I played around with how to represent the nebula and whether it should include the "N" for Nebula or be a separate thing.

**Reply by WillieCubed** [reply]
I'm a fan of that color palette! I think one thing to keep in mind is that we'll need a general Project Nebula logo (singular logo with separate wordmark) in addition to logos for all of the projects under its umbrella.
- Nebula Web (key words: "schedule planner")
- Nebula Data Service (key words: "database")
- UTD Survival Guide (key words: "handbook", "guide")

We should try shooting for some sort of small visual motif that appears throughout all the logos so students know all of these tools are connected. Something like a Comet (UTD's motif) but maybe something like a star. Just a thought! 😄

**Comment by litiwu** [comment]
I sketched made a few of the sketches into concepts. I'm trying to figure out how I can make the nebula portion have a minimalistic look but not look like a paper clip. I think it would be better to not include the "N" in the logo and have the wordmark logo separate, but then I also have to figure out how to make the nebula recognizable by itself.

![Untitled_Artwork 2](https://user-images.githubusercontent.com/59784432/124062122-c57ebe00-da62-11eb-801f-0e4f686f522f.png)
![Untitled_Artwork 3](https://user-images.githubusercontent.com/59784432/124062125-c6afeb00-da62-11eb-9839-16776d2703e9.png)
![Untitled_Artwork 4](https://user-images.githubusercontent.com/59784432/124062126-c7488180-da62-11eb-969d-bd9c21bee301.png)
![Untitled_Artwork](https://user-images.githubusercontent.com/59784432/124062127-c7e11800-da62-11eb-9f97-53c9325954ba.png)

**Comment by litiwu** [comment]
## Brainstorming No.2 
I didn't like where I was going with the first set of logo designs, so I took some time off and completely emptied my mind so I could go in a different direction. I took Willie's advice and started thinking more about how to work with shapes. I also wanted to incorporation a common element into the logos. 

![IMG_5962](https://user-images.githubusercontent.com/59784432/126101459-e70d68c2-5718-4c4b-bd84-44adfe610c6e.jpg)

These are the new sketches I came up with.

![IMG_5961](https://user-images.githubusercontent.com/59784432/126101431-e32f4615-a553-4d22-b785-c887dc062f54.jpg)

This is me trying out different ideas. I ended up liking having a wavy N inside a circle a lot.

## Drafts for the logos
![image](https://user-images.githubusercontent.com/59784432/126101733-68617f09-2dcd-4c5e-a888-7fee5322fe4a.png)
For this, I had just laid down the main shapes and colors for the logo. 

![image](https://user-images.githubusercontent.com/59784432/126101440-b6b6236d-9af3-4bc7-beb9-923ecdb343fa.png)
This is a more polished version, I also came up with a logo for the database.
I am thinking of changing the gradient of the N for the database one, maybe flip it upside down.

I think this is a good place to start for the logos, but it will probably go through many more changes! 

## Fonts
Ah. Picking fonts is actually the most difficult part for me. I am still browsing fonts to see what would work the best. It is hard to sample fonts and really feel how they will be presented to the users. I am still sampling them and hopefully will have a few picked tomorrow.

**Comment by litiwu** [comment]
# Font Sampling
Today I sampled some fonts, still deciding between them, but I'm just trying to capture the general feel of the fonts right now. 
![image](https://user-images.githubusercontent.com/59784432/126228349-4eb83b4e-070a-4e88-a44d-a5e24d38eb09.png)

**Reply by rajmeet2001** [reply]
Love the work Lisa! I think adding a futuristic text style would make the logo stand out!

**Comment by rajmeet2001** [comment]
# My Try at a Logo
When the idea for a Nebula logo came to be, I got pretty excited and jumped on some ideas for some logos. Because of some work outside, I never got a chance to bring the out to you guys. Here are couple of samples that I came up with. Please let me know if I can help out with the logo!
![1](https://user-images.githubusercontent.com/56010469/126549967-56905799-73d5-48a4-8be4-5817f89c209f.png)
![completion](https://user-images.githubusercontent.com/56010469/126549979-7b205b16-0672-4fc1-90f2-80d54628b5d0.png)

**Comment by litiwu** [comment]
![image](https://user-images.githubusercontent.com/59784432/131239175-f2b11d32-4bd4-4402-a30c-bfec12ba389e.png)

This is my updated branding idea! I made the logos have a cohesive color scheme.
Still deciding what the secondary color should be... blue or yellow? 
I made the wordmark logos as well.

**Reply by wbne** [reply]
I personally like blue and I think there should be a light and dark mode. If we have two modes then we can have blue be the dark mode secondary and yellow be the light mode secondary

**Reply by litiwu** [reply]
I love that idea! Here's the changes I made.
![image](https://user-images.githubusercontent.com/59784432/131406332-c35c3e0f-a879-4eba-b9cb-17c46fc163d3.png)

**Reply by taliadev** [reply]
I really dig the colors for both dark and light mode, but I'm not sure about the font choice, since the inconsistent stroke width might pose legibility issues at differing scales.
